### PR TITLE
Portuguese Brazilian April update

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Import Crowdin content buckets for Brazlian Portuguese from April

### CODE REVIEW

File: src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md [hunk 1]
```diff
@@ -3,10 +3,9 @@ title: Como criar um NFT (Segunda parte da série de tutoriais sobre NFT)
 description: Este tutorial descreve como criar um NFT na blockchain Ethereum usando nosso contrato inteligente e Web3.
 author: "Sumi Mudgil"
 tags:
-  - "NFTs"
   - "ERC-721"
-  - "Alchemy"
-  - "Solidity"
+  - "alchemy"
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md:7]:
```suggestion
  - "Alchemy"
```

File: src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md [hunk 2]
```diff
@@ -3,10 +3,9 @@ title: Como criar um NFT (Segunda parte da série de tutoriais sobre NFT)
...
+  - "solidity"
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md:8]:
```suggestion
  - "Solidity"
```

File: src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md [hunk 3]
```diff
@@ -74,17 +73,17 @@ Assim que você tiver criado sua conta:
...
 
-- Após fazer o upload, você verá as informações do arquivo na tabela da página Files. Você também verá a coluna CID. Você pode copiar o CID clicando no botão de cópia ao lado dele. Você pode ver seu ‘upload’ em: `https://gateway.pinata.cloud/ipfs/<CID>`. Você pode encontrar a imagem que usamos no IPFS [aqui](https://gateway.pinata.cloud/ipfs/QmarPqdEuzh5RsWpyH2hZ3qSXBCzC5RyK3ZHnFkAsk7u2f), por exemplo.
+- Após o upload, você verá as informações do arquivo na tabela da página "Arquivos". Você também verá a coluna CID. Você pode copiar o CID clicando no botão de cópia ao lado dele. Você pode ver seu ‘upload’ em: `https://gateway.pinata.cloud/ipfs/<CID>`. Você pode encontrar a imagem que usamos no IPFS [aqui](https://gateway.pinata.cloud/ipfs/QmZdd5KYdCFApWn7eTZJ1qgJu18urJrP9Yh1TZcZrZxxB5), por exemplo.
 
 Para os que aprendem de maneira mais visual, os passos acima são resumidos aqui:
 
-![Como fazer o upload de sua imagem para Pinata](https://gateway.pinata.cloud/ipfs/Qmcdt5VezYzAJDBc4qN5JbANy5paFg9iKDjq8YksRvZhtL)
+![Como fazer o upload de sua imagem para Pinata](./instructionsPinata.gif)
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md:82]: this image path doesn't exist

File: src/content/translations/pt-br/developers/tutorials/how-to-view-nft-in-metamask/index.md [hunk 4]
```diff
@@ -3,10 +3,8 @@ title: Como ver seu NFT na sua carteira (Parte 3/3 da série de tutorial sobre N
 description: Este tutorial descreve como visualizar um NFT existente no MetaMask!
 author: "Sumi Mudgil"
 tags:
-  - "NFTs"
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/tutorials/how-to-view-nft-in-metamask/index.md:6]: is it ok to remove these tags?

File: src/content/translations/pt-br/developers/tutorials/understand-the-erc-20-token-smart-contract/index.md [hunk 5]
```diff
@@ -6,11 +6,10 @@ tags:
   - "contratos inteligentes"
   - "tokens"
   - "solidity"
-  - "introdução"
   - "erc-20"
 skill: iniciante
 lang: pt-br
-published: 2020-04-05
+published: 05-04-2020
```
Comment (nhsz) [inline src/content/translations/pt-br/developers/tutorials/understand-the-erc-20-token-smart-contract/index.md:12]:
```suggestion
published: 2020-04-05
```

**Review by nhsz — CHANGES_REQUESTED** [review]
Some details to fix, left some questions

File: src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/translations/pt-br/developers/tutorials/how-to-mint-an-nft/index.md:7]:
```suggestion
```

I think we just get rid of the project name here

**Review by nhsz — APPROVED** [review]
LGTM

**Pull request merged**